Automattic/wp-calypso
# Support new bloganuary card. Enable bloganauary automatically in January

Supports showing the bloganuary my home card to more users and having it dismissed separately from the blogging prompts card. 

Discussed here pe7F0s-1iI-p2#comment-1840.

<img width="704" alt="Screen Shot 2023-12-14 at 12 46 07 pm" src="https://github.com/Automattic/wp-calypso/assets/22446385/dcbbb3a6-5d16-4c4b-a30b-ffae642eebfc">

## Testing Instructions

Apply diff D131976-code and you'll have to modify that diff to simulate it being January on the backend.

On the front end, apply `?flags=+bloganuary`.

You should see the blogging prompts card skinned for bloganuary.

You should be able to dismiss the card.

If "Bloganuary" is turned off, it should revert to showing the blogging prompts card if the site_intent===write. If the bloganuary card was dismissed, the blogging-prompts card will not be dismissed.

The card can be un-dismissed like so: `wp user-attributes undismiss_a_card --user_id=182004064 --blog_id=211923291 --card=home-section-bloganuary-blogging-prompt`
